### PR TITLE
Update episode crop data

### LINKED ISSUE #1: Episode 8: crop raster data
There are several things that need to be improved: 

- [ ] The data description should be updated;
- [ ] Both raster/vector will come from the open-source, therefore need to be cropped with bounding box 
- [ ] The Bounding box visualization should use a `non-filled` box. We should also bear in mind that some people may have difficulty reading the color;
- [ ] We should integrate original E6: reprojection 
- [ ] There need to be an introduction in raster -> raster reprojection?  We can try rasterize a vector (crop fields polygons), keep its crs, then reproject to the S2 image CRS
- [ ] The raster/PDOK_vector overlay should be visualized 
- [ ] Also teach the data exporting in this episode.

### PR DESCRIPTION
**Pull request by SarahAlidoost**
closes #1 

- [x] The data description should be updated;
- [x] Both raster/vector will come from the open-source, therefore need to be cropped with bounding box 
- [ ] The Bounding box visualization should use a `non-filled` box. We should also bear in mind that some people may have difficulty reading the color; 
       -  (I set `non-filled` box but regarding colors, I couldn't find a better way in this PR)
- [x] We should integrate original E6: reprojection 
- [x] There need to be an introduction in raster -> raster reprojection?  We can try rasterize a vector (crop fields polygons), keep its crs, then reproject to the S2 image CRS
- [x] The raster/PDOK_vector overlay should be visualized 
- [x] Also teach the data exporting in this episode.

### CODE REVIEW

File: _episodes/20-crop-raster-data.md [hunk 1]
```diff
@@ -212,4 +323,109 @@ raster_clip_wells.plot.imshow(ax=ax2)
 > > {: .language-python}
 > > <img src="../fig/20-crop-raster-dike-solution-07.png" title="Raster croped by buffer around dikes" width="512" style="display: block; margin: auto;" />
 > {: .solution}
-{: .challenge}
\ No newline at end of file
+{: .challenge}
+
+## Crop raster data using another raster data
+
+Imagine that we have two raster data, let's say the `scene` and `crop_fields` in
```
Comment (fnattino) [inline _episodes/20-crop-raster-data.md:330]:
```suggestion
Imagine that we have two raster datasets, let's say the `scene` and `crop_fields` in
```

File: _episodes/20-crop-raster-data.md [hunk 2]
```diff
@@ -212,4 +323,109 @@ raster_clip_wells.plot.imshow(ax=ax2)
...
+- method: Transverse Mercator
+Datum: World Geodetic System 1984
+- Ellipsoid: WGS 84
+- Prime Meridian: Greenwich
+~~~
+{: .output}
+
+We can see that these images are in different coordinate systems. Now, we can
+use `rioxarray.reproject_match()` function to crop `scene` image. This might
+take a few minutes, because the `scene` image is large.
```
Comment (fnattino) [inline _episodes/20-crop-raster-data.md:392]: This should already be loaded from the beginning of the episode as `raster` variable, right? Maybe we could use a more descriptive name (something like `visual`, or `true_color_image` ) and avoid re-downloading it at this stage?

**Review by fnattino — CHANGES_REQUESTED** [review]
Hi Sarah, looks great! As a general comment, I see this lesson download (twice) a true color image, which is quite large. We might keep it as it is for now, but think about ways to bypass the downloading of the full scenes (added comments below).

File: _episodes/20-crop-raster-data.md [hunk 3]
```diff
@@ -12,58 +12,147 @@ keypoints:
...
 
 ~~~
+import pystac
 import rioxarray
 
-# Load image and visualize
-raster = rioxarray.open_rasterio('data/S2_amsterdam.tif')
+# Open image and visualize it
+items = pystac.ItemCollection.from_file("search.json")
+raster = rioxarray.open_rasterio(items[1].assets["visual"].href) # Select a true color image
```
[the remark this answers is not in the document]
Comment (SarahAlidoost) [reply _episodes/20-crop-raster-data.md:42]: I added `raster.shape`. I noticed that plotting of the data takes time and not loading of the data. I amnot sure how to plot the "thumbnail". Let's discuss it tomorrow.

File: _episodes/20-crop-raster-data.md (hunk 2 above)
Comment (SarahAlidoost) [reply _episodes/20-crop-raster-data.md]: thanks, I replaced `raster` with `true_color_image` and also removed the re-loading part.

File: _episodes/20-crop-raster-data.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SarahAlidoost) [reply _episodes/20-crop-raster-data.md]: As discussed, we can't use the "thumbnail" image because it does not have a coordinate system.

**Comment by SarahAlidoost** [comment]
@fnattino thank you for reviewing. I addressed your comments. Please have a look.

**Review by fnattino — APPROVED** [review]
Great @SarahAlidoost, thanks a lot! Just added a minor suggestion, looks otherwise ready to me!

**Pull request merged**